website/docs/layouts/flex/index.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+
+export default class Index extends Component {
+  yourSidebarBooleanFlag = false;
```

Comment:
What is this for?

Reply:
I thought it was generated but the blueprint, but on a second look you're right, it comes from me copying the controller from the `AppFrame` docs, where it's actually used. I'll remove entirely the file, is not needed.